processor/src/range/mod.rs

```diff
@@ -80,15 +80,10 @@ impl RangeChecker {
         }
     }
 
-    // PUBLIC ACCESSORS
-    // --------------------------------------------------------------------------------------------
-
     /// Returns length of execution trace required to describe all 16-bit range checks performed
     /// by the VM.
     pub fn trace_len(&self) -> usize {
-        let (lookups_8bit, num_16bit_rows) = self.build_8bit_lookup();
-        let num_8bit_rows = get_num_8bit_rows(&lookups_8bit);
-        num_8bit_rows + num_16bit_rows
+        self.build_8bit_lookup().len
```

Comment:
nit: this seems to only be used in 1 test  now, so maybe we should mark it `#[cfg(test)]` and move it down down to the bottom under a "TEST HELPERS" section? Or at least grouped with the other public accessors?

Reply:
Done!